src/main/java/net/sf/jabref/external/DroppedFileHandler.java

```diff
@@ -345,7 +345,7 @@ public void stateChanged(ChangeEvent arg0) {
 
         linkInPlace.setText(Localization.lang("Leave file in its current directory."));
         copyRadioButton.setText(Localization.lang("Copy file to file directory."));
-        moveRadioButton.setText(Localization.lang("Move file to file directory."));
+        moveRadioButton.setText(Localization.lang("Move file to file directory") + '.');
```

Comment:
I would remove the first one and keep the other ones. Some languages also have a different sentence structure for questions...